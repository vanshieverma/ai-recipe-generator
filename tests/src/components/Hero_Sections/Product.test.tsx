import Product from "../../../../src/components/Hero_Sections/Product";
import { render } from '@testing-library/react'

it('The Product section renders', () => {
    const { container } = render(<Product resetPage={()=>({})}/>)
    expect(container).toMatchSnapshot()
})