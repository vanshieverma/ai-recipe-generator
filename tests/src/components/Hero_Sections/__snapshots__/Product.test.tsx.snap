// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Product section renders 1`] = `
<div>
  <div
    class="flex flex-col justify-center items-center px-4 md:px-8 lg:px-12"
  >
    <h1
      class="mb-2 text-4xl font-semibold text-gray-900 dark:text-white text-center"
    >
      Our Product
    </h1>
    <h2
      class="mt-6 mb-5 text-2xl leading-8 text-gray-600 text-center max-w-2xl"
    >
      Learn more about the amazing features of our product.
    </h2>
    <ul
      class="space-y-4 text-gray-500 list-inside dark:text-gray-400"
    >
      <li
        class="flex items-center text-lg"
      >
        <svg
          aria-hidden="true"
          class="w-3.5 h-3.5 me-2 text-green-500 dark:text-green-400 flex-shrink-0"
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M10 .5a9.5 9.5 0 1 0 9.5 9.5A9.51 9.51 0 0 0 10 .5Zm3.707 8.207-4 4a1 1 0 0 1-1.414 0l-2-2a1 1 0 0 1 1.414-1.414L9 10.586l3.293-3.293a1 1 0 0 1 1.414 1.414Z"
          />
        </svg>
        AI-powered recipe generation using your available ingredients.
      </li>
      <li
        class="flex items-center text-lg"
      >
        <svg
          aria-hidden="true"
          class="w-3.5 h-3.5 me-2 text-green-500 dark:text-green-400 flex-shrink-0"
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M10 .5a9.5 9.5 0 1 0 9.5 9.5A9.51 9.51 0 0 0 10 .5Zm3.707 8.207-4 4a1 1 0 0 1-1.414 0l-2-2a1 1 0 0 1 1.414-1.414L9 10.586l3.293-3.293a1 1 0 0 1 1.414 1.414Z"
          />
        </svg>
        Customized recipes based on dietary preferences and restrictions.
      </li>
      <li
        class="flex items-center text-lg"
      >
        <svg
          aria-hidden="true"
          class="w-3.5 h-3.5 me-2 text-green-500 dark:text-green-400 flex-shrink-0"
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M10 .5a9.5 9.5 0 1 0 9.5 9.5A9.51 9.51 0 0 0 10 .5Zm3.707 8.207-4 4a1 1 0 0 1-1.414 0l-2-2a1 1 0 0 1 1.414-1.414L9 10.586l3.293-3.293a1 1 0 0 1 1.414 1.414Z"
          />
        </svg>
        User-friendly interface to easily add ingredients and generate recipes.
      </li>
      <li
        class="flex items-center text-lg"
      >
        <svg
          aria-hidden="true"
          class="w-3.5 h-3.5 me-2 text-green-500 dark:text-green-400 flex-shrink-0"
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M10 .5a9.5 9.5 0 1 0 9.5 9.5A9.51 9.51 0 0 0 10 .5Zm3.707 8.207-4 4a1 1 0 0 1-1.414 0l-2-2a1 1 0 0 1 1.414-1.414L9 10.586l3.293-3.293a1 1 0 0 1 1.414 1.414Z"
          />
        </svg>
        Option to save, rate, and share your favorite recipes.
      </li>
    </ul>
    <button
      class="mt-10 rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
    >
      Back to Home
    </button>
  </div>
</div>
`;
